test/map.spec.js

```diff
@@ -277,6 +277,58 @@ test('getTileTraversabilityInCells: returns correctly with nontraversable tile,
 });
 
 
+test('getTileTraversabilityInCells: returns correctly with angled wall 1, CPTL=4', t => {
```

Comment:
So, the way we treat circular objects is if any part of the cell intersects with the NT part of the object, that cell is a 0. If we stick to that convention (which I think we should), this test should actually return:

```
[1, 0, 0, 0],
[1, 1, 0, 0],
[1, 1, 1, 0],
[1, 1, 1, 1],
```

This is especially important for low CPTL. it looks like this implementation requires the entire cell to be non-traversable.